packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -341,19 +374,20 @@ const DialpadContainer = (props: {
     >
       <TextField
         styles={concatStyleSets(textFieldStyles(theme), props.styles?.textField)}
-        value={textFieldValue ? textFieldValue : formatPhoneNumber(plainTextValue)}
+        componentRef={inputBoxReference}
```

Comment:
Just curious what this reference is used for?

Reply:
ah leftover from something I was trying I will remove it